src/models/transactions.js

```diff
@@ -1,7 +1,28 @@
-const { Schema, model } = require("mongoose");
+const { Schema, model } = require('mongoose');
 
-const transaction = new Schema({}, { versionKey: false, timestamps: true });
+const transaction = new Schema(
+  {
+    type: {
+      type: String,
+      enum: ['-', '+'],
+    },
+    category: {
+      type: String,
```

Comment:
Nie wiem czy nie powinna pojawić się lista kategorii (jakiś array lub coś - bo jeśli ma być wyszukiwanie po kategoriach to trzeba ograniczyć chyba tylko do istniejących)